### ISSUE TITLE
Start G7 sensor session within loop (without G7 app - dangerous international travel restrictions)

### ISSUE THREAD

**Issue by ianjorgensen** [LoopKit/G7SensorKit#27]
I was unfortunate enough to have to re-install the dexcom g7 app while traveling abroad only to find out that dexcom doesn't not allow this. I was told I would be unable to get readings for the remainder of my holiday in Greece, there is no good workaround and dexcom official response was to have me go back to a g7 supported country to activate the app. 

Have you looked into directly initiating a g7 sensor within loop? 

Location spoofing didn't work, but I was able to circumvent the restriction in the G7 app by creating a Greek account online (not via the app) then on my iPhone under developer settings I enabled network limit condition and set it to 100% loss. This gave me the apps tcp timeout time to setup the sensor after many tries of restarting the app to get further in the process. Luckily the app saves the setup state after restarts. It's not a pretty workaround, and a g7 app update might make it so you can't do this anymore. Also I'm not sure the dexcom website would let you create an account for every country. 

Knowing I can't safety travel outside the list of dexcom g7 supported countries is a burden I, and am sure others, would love to live without. 

💚

**Comment by jlmgithub** [comment]
Just a comment about the issues of Dexcom geofencing and app failures: **https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfmaude/search.cfm**
You can report these events to the US FDA on the MAUDE database:  Manufacturer and User Facility Device Experience.
The MAUDE database houses medical device reports submitted to the FDA by mandatory reporters (manufacturers, importers, and device user facilities) and voluntary reporters such as healthcare professionals, patients, and consumers.

Dexcom has been aware of these issues for over a year but has taken no steps to share the problem and provide potential solutions to its G7 customers.

**Comment by ianjorgensen** [comment]
Indeed. I have submitted the issue to the danish medical device authorities https://portal.dkma.dk/indberetningmuborgerpaaroerende?sc_lang=da

**Comment by ianjorgensen** [comment]
To add some disturbing background, the app gave me an error message asking me to uninstall the app while trying to pair a new sensor. 

There is basically no guarantee you will be able to get cgm reading while traveling unless you have a standalone reader.

**Comment by motinis** [comment]
This issue has been fixed as of Dexcom G7 App 2.7 I believe

**Comment by marionbarker** [comment]
Please close this Issue by scrolling to the bottom of the screen at GitHub and tapping on "Close issue"

BR/Marion

**Comment by jlmgithub** [comment]
I get the following message:  You do not have permission to close this issue.

Joanne Milo

www.TheSavvyDiabetic.com

T1D to 100 Study Group, founder
Loop and Learn (So Cal Loopers) founder
#DeDoc°
DiabetesMine D-Data/Innovation Summit
Top 50 Best Type 1 Diabetes Blogs/Feedspot
